Makefile

```
@@ -27,6 +27,11 @@ DIST_VER=$(shell git tag | sort -r --version-sort | head -n1)
 -include config.override.mk
 include config.mk
 
+# In case DIST_VER is not assigned, fallback to using the latest semver tag available
+ifdef DIST_VER
```

It will override when DIST_VER is assigned with ifdef
```suggestion
ifndef DIST_VER
```

---

Rebased and pushed changes